### PR TITLE
added freight calibration package

### PR DESCRIPTION
**Pull request by narora1**

### CODE REVIEW

**Comment by narora1** [comment]
I tested this on freight 28, will test on freight 0 in a bit

File: freight_calibration/scripts/calibrate_freight [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    launch_xml = etree.parse("/etc/ros/%s/robot.launch" % rosdistro)
+    for child in launch_xml.getroot():
+        if child.tag == "param" and child.get("name") == "calibration_date":
+            return child.get("value")
+    return "Calibration date not found in robot.launch"
+
+if __name__ == "__main__":
+    # Parse the arguments
+    parser = argparse.ArgumentParser(description="Calibrate the robot, update files in /etc/ros")
+    parser.add_argument("--base", help="Captyure base calibration data", action="store_true")
```
Comment (mikeferguson) [inline freight_calibration/scripts/calibrate_freight]: type -- "Capture"

Comment (mikeferguson) [inline freight_calibration/scripts/calibrate_freight]: how fitting that I would have a typo for "typo"....

File: freight_calibration/scripts/calibrate_freight [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        # Reset the robot.launch to factory defaults
+        copy_file(os.path.join(bringup, "launch/freight.launch"), etc_launch)
+        restart = True
+
+    if args.base:
+        if subprocess.call(["rosrun", "robot_calibration", "calibrate_base"]) != 0:
+            print("Failed to run calibrate_Base")
+            exit(-1)
+
+    if args.install or args.arm or args.base:
```
Comment (mikeferguson) [inline freight_calibration/scripts/calibrate_freight]: still has .arm

File: freight_calibration/scripts/calibrate_freight (hunk 1 above)
Comment (mikeferguson) [inline freight_calibration/scripts/calibrate_freight]: also -- "base" is for the odometry/imu stuff -- it seems we are missing a "base_camera" or similar, to actually run capture.launch (as we do with the "arm" parameter in capture_robot in fetch_calibration

**Comment by narora1** [comment]
@mikeferguson done. I added a base_camera

File: freight_calibration/scripts/calibrate_freight [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+            rp = rospkg.RosPack()
+            bringup = rp.get_path("freight_bringup")
+        except:
+            print("\nCould not find freight_bringup package, is your ROS path correct?\n")
+            exit(-1)
+        # Reset the robot.launch to factory defaults
+        copy_file(os.path.join(bringup, "launch/freight.launch"), etc_launch)
+        restart = True
+
+    if args.base_camera:
```
Comment (mikeferguson) [inline freight_calibration/scripts/calibrate_freight]: base_camera doesn't appear to be in the parser.add_argument calls

**Comment by narora1** [comment]
@mikeferguson sorry missed that. I fixed it now.

**Pull request merged**